### PR TITLE
update white list of assets allowed domains

### LINKED ISSUE #327: Refused to load ... because it violates the following Content Security Policy directive
I'm seeing lots of the following in the console `[Report Only] Refused to load [x] because it violates the following Content Security Policy directive:`

### PR DESCRIPTION
**Pull request by yotammanor**
fix #327.

Changes introduced:
- add custom assets domain from env vars (set to a.okfn.org).
- update domains being used in CSP_IMG_SRC and CSP_FONT_SRC.

### CODE REVIEW

File: foundation/settings.py [hunk 1]
```diff
@@ -363,6 +365,8 @@ def _parse_email_list(varname):
         asset_hosts.append('https://%s' % AWS_S3_CUSTOM_DOMAIN)
     else:
         asset_hosts.append('https://s3.amazonaws.com')
+    if CUSTOM_ASSETS_DOMAIN:
+        asset_hosts.append('https://%s' % CUSTOM_ASSETS_DOMAIN)
```
Comment (brew) [inline foundation/settings.py]: Will `CUSTOM_ASSETS_DOMAIN` always never include the protocol prefix?

Comment (yotammanor) [inline foundation/settings.py]: @brew I was following the pattern of AWS_S3_CUSTOM_DOMAIN, on https://github.com/okfn/website/pull/367#discussion-diff-89797742R365

Comment (yotammanor) [inline foundation/settings.py]: @brew I'm merging as it is, but let me know what you think and I'll change things in a different PR if deemed necessary.

Comment (brew) [inline foundation/settings.py]: I don't know, which is why I asked :) If you think it's not a problem, then go for it.

Comment (yotammanor) [inline foundation/settings.py]: I think I'll open a separate issue for both of those env var URLs. Thanks.

**Pull request merged**